### DISCUSSION TITLE
Print .PNG files from SD card

### DISCUSSION THREAD

**Discussion by ELECTRONOOBS** [Bodmer/TFT_eSPI#2845] [Compatible support libraries]
Till now the library works great for me with different screens. I was working with the ILI9341 for example and using the jpeg from SD card example works ok. That example is using the jpeg decode library bur I can see there is also a PNG decode library. Bur there is no SD card example for PNG, only flash stored files. I was wondering if is there an example of opening a PNG dile from tbe SD card, decode it and print it on the screen just as with the jpeg example. Thank you!!!

**Comment by Bodmer** [comment]
You could try chacging the[ LittleFS example](https://github.com/Bodmer/TFT_eSPI/tree/master/examples/PNG%20Images/LittleFS_PNG).

Change[ these lines](https://github.com/Bodmer/TFT_eSPI/blob/master/examples/PNG%20Images/LittleFS_PNG/LittleFS_PNG.ino#L13C1-L14C25) to
```
#include <SD.h>
#define FileSys SD
```
Other changes may be required but tat would be a good starting point.

The example here may also help:
https://github.com/bitbank2/PNGdec/blob/master/examples/sdcard_slideshow/sdcard_slideshow.ino
